apps/web/package.json

```diff
@@ -76,6 +76,7 @@
     "less": "^4.1.0",
     "lodash.capitalize": "^4.2.1",
     "lodash.clonedeep": "^4.5.0",
+    "lodash.debounce": "^4.0.8",
```

Comment:
I'm using the debounce logic for the search and transaction id inputs... more about it later
the size of this lib is ~1kB gzip